styles/components/_gameModeName.scss

```diff
@@ -0,0 +1,21 @@
+.mode-info {
+  background-color: $color-primary;
+  max-width: 70vw;
```

Comment:
https://www.sitepoint.com/css-viewport-units-quick-start/ 👀 

Start at "Things to keep in mind"